experimental/Schemes/AffineSchemes.jl

```diff
@@ -415,7 +415,7 @@ function issubset(
     Y::AbsSpec{BRT, RT}
   ) where {BRT, RT<:MPolyQuoLocalizedRing{<:Any, <:Any, <:Any, <:Any, <:MPolyPowersOfElement}}
   R = base_ring(OO(X))
```

Comment:
Why do you replace base_ring(OO(Y)) by ambient_ring(Y) for Y, but not for X? (here and possibly further occurrences)

Reply:
In the end they are equivalent (at least I expect so). But for the sake of consistency I also removed the rest.